### ISSUE TITLE
Shift+F6 key does not work for Server edition

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#7474]
### System details

    RStudio Edition : Server
    RStudio Version : v1.4.647
    OS Version      : Windows 10 with WSL2, Chrome v84.0.4147.105
    R Version       : ANY

### Steps to reproduce the problem

1. After logged in RStudio Server, press F6 to go to the next pane.
2. Press Shift+F6 and see if the focus returns to the previous pane.

### Describe the problem in detail

This feature was originally proposed via #6882, and closed via #7408.

Two issues I have found are ..:

1. Shift+F6 does not work for focusing panes backwords.
1. As explained in #6882, it would be ideal to add `CTRL` modifier key in addition to F6 for Server edition because F6 is often taken by browsers for focusing address bar. In sum, we may want to keep F6/Shift+F6 for Desktop edition; Ctrl+F6/Ctrl+Shift+F6 for Server edition. This is how other apps (e.g., Slack) do for Desktop and Web versions.

### Describe the behavior you expected

1. Shift+F6 has to work for Server  edition.
1. If possible, please add Ctrl modifier key for Server edition.

- [X] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [ ] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [X] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by melissa-barca** [comment]
Hi @jooyoungseo I believe the issue with server mode is not that F6 doesn't work, but it currently requires the user to have focus within one of the panes to do anything. Once you navigate to a pane, F6 should work properly. I will update the feature so that if there is not focus in a pane, focus defaults to the top left pane when F6 is selected. Let me know if this sounds correct to you.

Regarding adding the Ctrl key, we currently use "Ctrl + F6" for `activateFindInFiles` and "Ctrl + Shift + F6" for `layoutZoomTutorial`. I'd like to hear @gtritchie's thoughts on if we should replace these. This also makes me wonder if there are more commands that overlap with browser defaults.

**Comment by jooyoungseo** [comment]
Thanks for your response, @melissa-barca.

I am not sure whether this is because I am using a screen reader, but pressing `Shift+F6` two times puts my focus to the browser addressbar while `F6` works really well.

In short, the forward (F6) navigation works; backword (Shift+F6) navigation does not work (please press Shift+F6 at least two times).

As for the key conflict against the existing RStudio keys, I think replacing them with a new key would cause unexpected confusion.

This is not ideal, but I would suggest that we stick with F6/Shift+F6 as is.

Fortunately, there is another addressbarFocus key which is Alt+D on Windows, but I have no idea about Mac/Linux's alternative keys.

Yes, we need @gtritchie's advice and insight.

**Comment by gtritchie** [comment]
I'm OOO today (barely). I always using Ctrl+L or Cmd+L to focus the address bar in browsers.

We do try to avoid conflicts with browser shortcuts, but have made several exceptions over time.

Will try it out later (on Windows, I don't think I tried the Shift+F6 work on Windows, only Mac).

**Comment by R-Hannibal** [comment]
Marking this issue verified due to the not reproducible resolution.
